behaviors/v1_1/management-chain.md

```diff
@@ -0,0 +1,237 @@
+# Management Virtual Chain
+> Provide a system wide source of truth across virtual chains.
+> Initial feature abstracts the governance process results (occurs on the Ethereum network) on an Orbs Validator Node level - across its virtual chains' instances.
+
+## Main Participating components
+* New CrossChain-API service.
+* New VirtualChainConnector service.
+* Management chain query SDK.
+* Off-chain data (Non-deterministic) check as part of the block validation. 
+* Updated receipt format to include off-chain data.
+* Elected validators validity check as part of the PreOrder check.
+* Custom system contracts deployment per virtual chain type.
+* Elected Validators contract.
+* Updated RequestOrderedCommittee flow.
+
+## Design principles
+* Assumptions:
+    * "Trust" holds inside the Node (across its virtual chains instances)
+    * Reference Validators set recorded on the virtual chain (VC) holds during VC genesis and until the state is updated. (Assumes a progress of the VC)
+      * Read an external data source not under consensus may lead to network split.
+    * Current Validators set is "honest" (Quorum wise) during the election transition (PoS property). 
+
+* Hierarchical VCs design: Management-VC and dependent Virtual Chains (Consumer-VC).
+* Custom system contracts deployment for the different VC flavors (Management-VC, Consumer-VC, Private-VC). 
+    * Currently, this only holds for `_ElectedValidators` system contract (interchangeably coined "Elected Validators Provider"). Where there are two types of providers: remote and local.
+
+* Management-VC
+    * Dedicated VC for cross system source of truth
+    * Used for elected validators, system config (future - subscription, and other cross VCs interaction).
+    * Holds multiple last states close to tip for state queries by block height (facilitates the inner node sync across its VC instances and in turn the underlying agreement).
+    * Holds the `_Election` contract for Ethereum election information.
+        * The `_Election` contract takes the role of the "Elected Validators Provider" on the Management-VC.
+    * Holds the `_Provision` system contract which provides information about the different Consumer-VCs subscription and initial config - includes the Reference Validators Set (currently not supported).
+
+* Consumer-VC
+    * Continuously sync system config.
+        * Update logic - depends on the record expiration.
+        * Read logic - holds until it is overwritten.
+    * Close to real-time information retrieval, with small time phase shift (to support agreement - only for small inner node syncing issues (including its other VCs instances))
+    * Maintains a single cache entry per `block.timestamp` (reference time -> offChain data). This implies the same context for the entire block.
```

Comment:
what's a reasonable eviction policy for this cache? is this left for the developers to decide?

Reply:
Only one value may be cached, and it is only valid for the current block so no eviction policy is required. the value will be cached for the duration of processing all transactions in a single block.